edu_edfi_airflow/callables/snowflake.py

```diff
@@ -26,6 +28,11 @@ def insert_into_snowflake(
     # Retrieve the database and schema from the Snowflake hook.
     database, schema = airflow_util.get_snowflake_params_from_conn(snowflake_conn_id)
 
+    logging_string = f"Inserting the following values into table `{database}.{schema}.{table_name}`\nCols: {columns}\n"
```

Comment:
Because `SnowflakeHook.insert_rows()` truncates the row outputs, we needed to manually display the outputs to improve visibility.